packages/destination-actions/src/destinations/customerio/deleteObject/index.ts

```diff
@@ -13,7 +13,7 @@ const action: ActionDefinition<Settings, Payload> = {
       description: 'An object ID used to identify an object.',
       type: 'string',
       default: {
-        '@path': '$.context.groupId'
+        '@path': '$.groupId'
```

Comment:
Hi @sydneycollins-cio - would it not make sense to do this following?

```suggestion
        '@path': '$.properties.objectId'
```